### DISCUSSION TITLE
[Support] Font doesn't look right on scaled images

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#3078] [Q&A] [answered]
## Environment

- Minimal Mistakes version: 4.24.0
- Ruby gem or remote theme version: `ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x64-mingw32]`
- Jekyll version:  `jekyll 4.2.0`
- Git repository URL: [jekyll.zip](https://github.com/mmistakes/minimal-mistakes/files/6872651/jekyll.zip)
- GitHub Pages hosted (if yes provide URL to site): Self Hosted
- Operating system: Windows 10 21H1 19043.1110

## Expected behavior

I am using Minimal Mistakes for my Jekyll site. I am currently hosting it by myself before publishing it. I used the Feature Row for making some pages of my website visible on the homepage. Because the image was too big, I resized it on `\_sass\minimal-mistakes\_archive.scss`. I thought the font would line up correctly with the picture but it doesn't.

## Steps to reproduce the behavior

* Edit Line 301 of `\_sass\minimal-mistakes\_archive.scss`
* Set Width to 40%
* Use `{% include feature_row type="left" %}` on the page
* Use Splash Layout
* Refresh page

## Other

![image](https://user-images.githubusercontent.com/52204908/126872946-5e7fb50e-73ec-48df-830d-76d96010fceb.png)

**Comment by mmistakes** [comment, marked as answer]
The margin above the headlines is intentional.
If you'd like to reduce that spacing you'll need to [customize the CSS](https://mmistakes.github.io/minimal-mistakes/docs/stylesheets/#customizing) by adding something like `margin-top: 0;` in the `.archive__item-title` selector.

Relevant file/line: https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_archive.scss#L247

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/52204908/126998267-d4cb474a-3a17-4e93-9057-586f76a8892a.png)
Thanks, worked for me.
